10up/Ad-Refresh-Control
# upkeep/node: Upgrade node to v20

### Description of the Change



Closes #150

- Upgrade to node v20.
- Cleans up NPM dependencies.
- Modernizes build process by using `@wordpress/scripts`.

### How to test the Change


Use this plugin to test and verify - https://gist.github.com/peterwilsoncc/f4b376c2c34084701fa5821d97a296da

### Changelog Entry


> Changed - Clean up NPM dependencies and update node to v20


### Credits

Props @Sidsector9 


### Checklist:
